packages/babel-preset-stage-0/index.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  presets: [
+    require("babel-preset-stage-1")
+  ],
+  plugins: [
+    require("babel-plugin-syntax-do-expressions"),
+    require("babel-plugin-syntax-function-bind"),
+    require("babel-plugin-transform-do-expressions"),
```

Comment:
When you enable transforms it also enables the syntax so we don't need to double up.